runtime/lua/vim/lsp.lua

```diff
@@ -663,6 +660,10 @@ end
 --- - exit_timeout (number, default 500): Milliseconds to wait for server to
 --        exit cleanly after sending the 'shutdown' request before sending kill -15.
 --        If set to false, nvim exits immediately after sending the 'shutdown' request to the server.
+---
+---@param root_dir string To be deprecated in 0.7. Directory where the LSP
```

Comment:
```suggestion
---@param root_dir string To be removed in 0.7. Directory where the LSP
```

Not sure which way you want it,  but the docs here don't match the message further below